tests/src/integration/ccm/bridge.cpp

```diff
@@ -1491,7 +1491,7 @@ CCM::Bridge::generate_create_updateconf_command(CassVersion cassandra_version) {
     updateconf_command.push_back("enable_scripted_user_defined_functions:true");
   }
 
-  if (cassandra_version >= "4.0.0") {
+  if (cassandra_version >= "4.0.0" && !is_dse()) {
```

Comment:
A small fix to get DSE working for integration tests.  IIRC @weideng1 has something similar to this in another PR.